extensions/php/package.json

```diff
@@ -15,6 +15,14 @@
 			"aliases": [ "PHP", "php" ],
 			"mimetypes": ["application/x-php"],
 			"configuration": "./language-configuration.json"
+		},
+		{
+			// For reference by services requiring PHP syntax highlighting for partial
+			// code missing script start/end tags.
+			"id": "phplanguage",
+			"aliases": [ null ], // unsupported: prevents language from appearing in Language Mode list
```

Comment:
Indeed, it could be! At the moment, I opted to simply opted to light up the [hack that was already in use](https://github.com/mousetraps/vscode/blob/i14166/src/vs/editor/common/services/languagesRegistry.ts#L170) because there were enough things that depended on the existence of an array, but on closer look, they already all have isArray checks, so agreed that "null" would be preferable.

Or.... I suppose one could argue that an array is nice, because you could also add additional invisible aliases? Not *actually* arguing it... just saying that one *could* do such a thing. 😜